CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - **CUMULUS-836** - `@cumulus/api` added IAM roles required by the NGAP 2.0 AMI. The NGAP 2.0 AMI runs a script `register_instances_with_ssm.py` which requires the ECS IAM role to include `ec2:DescribeInstances` and `ssm:GetParameter` permissions.
 
 ### Fixed
-
+- **CUMULUS-839** - SyncGranule 'collection' is now an optional config parameter
```

Comment:
This is in the wrong spot, since it's under 1.9.1. Move this comment up to the unreleased section.